extensions/amp-story/1.0/_locales/en.json

```diff
@@ -250,5 +250,29 @@
   "89": {
     "string": "Your response will be sent to",
     "description": "Text displayed to users after clicking a button that reveals a disclaimer, telling them more about where their user data will be stored, after they interact with a poll or quiz."
+  },
+  "90": {
+    "string": "Next story",
+    "description": "Label for a button that advances to the next element in the carousel."
+  },
+  "91": {
+    "string": "Next page",
+    "description": "Label for a button that advances to the next page of the story."
+  },
+  "92": {
+    "string": "Replay",
+    "description": "Label for a button that replays the story."
+  },
+  "93": {
+    "string": "Previous page",
+    "description": "Label for a button that returns the user to the previous page of the story."
+  },
+  "94": {
+    "string": "Close bookend",
+    "description": "Label for a button that closes a dialog and returns the user to the article."
```

Comment:
Nit: plz add the bookend description as you have on the `95`, it's super helpful

Reply:
> Nit: plz add the bookend description as you have on the `95`, it's super helpful

# - [ ] ~~~~